libbeat/processors/dissect/dissect_test.go

```diff
@@ -0,0 +1,282 @@
+package dissect
+
+import (
+	"encoding/json"
+	"flag"
+	"io/ioutil"
+	"regexp"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var export = flag.Bool("dissect-tests", false, "export dissect tests to JSON.")
```

Comment:
As this test is in the `dissect` package I wonder if this flag will show up globally. Probably not as _test files should not be compiled, but better double check.

Reply:
Its only scoped to this part and won't be present in the end binary.